### ISSUE TITLE
chore: add meeting notes to sig-documentation

### ISSUE THREAD

**Issue by lechenghiskhan** [akash-network/community#49]
Add meeting notes to sig-documentation. 
This should be completed shortly (speaking with a community member to take the note bounties).

**Comment by figurestudios** [comment]
Let me know if this is as you wanted it, was hard to follow the discussion when it got off-topic at points:

https://github.com/akash-network/community/pull/64

The agenda was already in place, so I only added notes for the meeting.

@brewsterdrinkwater @lechenghiskhan
